packages/design-system/src/components/text-field.tsx

```diff
@@ -32,28 +35,43 @@ export const TextField = styled("input", {
     fontFamily: "$untitled",
     color: "$hiContrast",
   },
+  "&::placeholder": {
+    color: "$slate9",
+  },
+
+  "&:disabled": {
+    color: "$slate8",
+    // @todo: This is not working now bue to cursor variant. Is it needed?
```

Comment:
not sure what you mean

Reply:
I removed the `cursor` variant since it wasn't used so I think this can be ignored.